### PR TITLE
refactor(tooltip): rewrite with hook

### PR DESCRIPTION
**Pull request by hengkx**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  refactor with hook         |
| 🇨🇳 Chinese |  用 react hook 重构         |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/slider/SliderTooltip.tsx [hunk 1]
```diff
@@ -18,9 +18,7 @@ export default function SliderTooltip(props: TooltipProps) {
     }
 
     rafRef.current = window.requestAnimationFrame(() => {
-      if (tooltipRef.current && (tooltipRef.current as any).tooltip) {
-        (tooltipRef.current as any).tooltip.forcePopupAlign();
-      }
+      tooltipRef.current.forcePopupAlign();
```
Comment (hengkx) [inline components/slider/SliderTooltip.tsx:21]: 我没办法了 这个 方法要怎么 才能测到

Comment (zombieJ) [reply components/slider/SliderTooltip.tsx:21]: 搜一下 jest requestAnimationFrame 看看有没有相关内容，印象里的确有点绕弯弯。

**Pull request merged**